packages/widget_3d_tiles/src/index.js

```diff
@@ -17,19 +58,27 @@ export class C3DTiles extends itownsWidget.Widget {
    * @param {string} options.layerContainerClassName - class name of the layer container div
    * @param {string} options.urlContainerClassName - class name of the layer container div
    * @param {string} options.c3DTFeatureInfoContainerClassName - class name of the c3DTFeatureInfo container div
+   * @param {boolean} options.displayExistingLayers - whether the existing layers should be listed in the UI or not (default is True)
    */
   constructor(view, options = {}) {
     super(view, options, DEFAULT_OPTIONS);
+    // Available layers are optionnaly listed in the UI. Inhibiting this display
+    // allows for an alternative usage of other widgets with a similar purpose
+    // but different feature e.g. @ud-viz/widget_layer_choice.
+    if (typeof options.displayExistingLayers == 'undefined') {
```

Comment:
The operator typeof is not required in this condition: 
`if(options.displayExistingLayers == undefined)`
or
`if(!options.displayExistingLayers)`